First pass at scraping solar spell

dkim0526 opened the pull request:
##### Description

This PR is to add sushi chef for `solarspell.org` to import content from `solarspell.org` into Kolibri studio. 
- update  in `sushichef.py`
- add a thumbnail image 
- update some readme content.

dkim0526 committed:
First pass at scraping solar spell

This commit contains changes of
scraping content from solarspell.org
into Kolibri studio.

- Update sushi chef code
- Update readme content

jayoshih commented on sushichef.py:
```diff
@@ -45,35 +49,84 @@ class MyChef(SushiChef):
+                source_id=source_id,
+                title=video_title,
+                files=[video_file],
+                license=CHANNEL_LICENSE
```
Nothing to be done about it yet, but will eventually need to be updated on a per-item basis (will let you know when this can be done). Might also need to add copyright_holder field depending on the license

jayoshih commented on sushichef.py:
```diff
@@ -45,35 +49,84 @@ class MyChef(SushiChef):
+        # Check if it is mp4 file
+        if source_id.endswith(".mp4"):
+            video_info = attribute.find("a")
+            video_title = str(video_info.string)
```
Could you change this to `video_title, _ext = splitext(str(video_info.string))` to remove the .mp4 ending?

jayoshih commented on sushichef.py:
```diff
@@ -45,35 +49,84 @@ class MyChef(SushiChef):
+            existingNode.add_child(video_node)         
+
+        # Check if it is a directory
+        if source_id.startswith("dir"):
```
Super minor, but might be good to change to `elif` for performance

jayoshih commented on sushichef.py:
```diff
@@ -45,35 +49,84 @@ class MyChef(SushiChef):
 
+            new_end_point = replace_all(title, replacements)
+            new_end = endpoint + "{}/".format(new_end_point)
+            scrape_content(new_end, channel, topic_node)
```
Could you also have an `else` statement for any unrecognized formats and use `LOGGER.error` to log it? Might be useful for discovering any edge cases

jayoshih reviewed (changes requested):
Just a few super minor tweaks and should be good to go! Great work with commenting and organization - super easy to read

dkim0526 committed:
Update some changes from feedbacks received

- Use `elif` for for performance improvement,
- Add `logger` to record unrecognized formats
by the sushi chef.
- Use `splitext` to exclude non `mp4` in video titles.

Note: code still needs some changes on license
